app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -1,102 +1,57 @@
-import React, { useState, useEffect } from "react";
+import React, { useState } from "react";
 
 import EmptyNotesListImage from "images/EmptyNotesList";
-import { Delete } from "neetoicons";
-import { Button, PageLoader } from "neetoui";
-import { Container, Header, SubHeader } from "neetoui/layouts";
+import { Button } from "neetoui";
+import { Container, Header } from "neetoui/layouts";
 
-import notesApi from "apis/notes";
 import EmptyState from "components/Common/EmptyState";
 
-import DeleteAlert from "./DeleteAlert";
-import NewNotePane from "./Pane/Create";
-import Table from "./Table";
+import { NOTES_CARD_DATA as notesData } from "./constants";
```

Comment:
This is unnecessary. Literal constants should be named in uppercase snake_case.

Is there a specific reason why you felt this ought to be converted to camleCase?

- Secondly, no need to suffix with the `Data` keyword unless there is a conflicting variable in the scope. Here I believe you are trying replace the `notes` state with `notesData`. If that's the case then calling it `notes` is more appropriate.

Reply:
I would like to clarify whether you pointed out issue with using an alias for importing the NOTES_CARD_DATA or for using camelCase? If it was issue with aliasing, I didn't know this was a wrong approach.

Reply:
I would like to clarify whether you pointed out issue with using an alias for importing the NOTES_CARD_DATA or for using camelCase? If it was issue with aliasing, I didn't know this was a wrong approach. I will keep this in mind.

 Regarding the naming convention, there was already a state named notes so when I hardcoded the constants, in order to avoid conflict I changed it to notesData at the beginning.

Reply:
> If it was issue with aliasing, I didn't know this was a wrong approach. I will keep this in mind.

👍 

>there was already a state named notes so when I hardcoded the constants, in order to avoid conflict I changed it to notesData at the beginning.

My point was you had removed that `notes` state. Thus it doesn't make sense now to name the constant as `notesData` whereas you could name it as `notes` itself.